api_sentry.py

```diff
@@ -0,0 +1,208 @@
+import os
+import sys
+
+import pandas as pd
+
+
+from lib.sentry_conn import APIClient
+
+# TODO: Database
+from database import (
+    Database,
+    ReportSentryIssues
+)
+
+
+class Sentry:
+
+    def __init__(self):
+        try:
+            self.client = APIClient(os.environ['SENTRY_HOST'])
+            self.client.api_token = os.environ['SENTRY_API_TOKEN']
+            self.organization_slug = \
+                os.environ['SENTRY_ORGANIZATION_SLUG']
+            self.project_id = os.environ['SENTRY_PROJECT_ID']
+            self.project_slug = 'mozilla'
+        except KeyError:
+            print("ERROR: Missing testrail env var")
+            sys.exit(1)
+
+    # API: Issues
+    # Question: Should we just list the for_review issues?
+    #           Any other types of issues?
+    # Only the unassigned issues past day sorted by frequency:
+    # organization/mozilla/issues/
+    # ?limit=10&query=is:for_review&sort=freq&statsPeriod=1d
+    def issues(self, release_version='137.0'):
```

Comment:
We don't need to hard code the release, this info comes in the response json for each issue. We get all the issues and then filter the release when we digest the data